test/NuGet.Core.FuncTests/Dotnet.Integration.Test/MsbuildIntegrationTestFixture.cs

```diff
@@ -114,7 +253,7 @@ private static void CopyFromTemplate(string projectName, string args, string wor
                 Path.Combine(workingDirectory, projectName + ".csproj"));
         }
 
-        internal void CreateDotnetToolProject(string solutionRoot, string projectName, string targetFramework, string rid, string source, IList<PackageIdentity> packages, int timeOut = 60000)
+        internal void CreateDotnetToolProject(string solutionRoot, string projectName, string targetFramework, string rid, string packageSource = null, IList<PackageIdentity> packages = null, int timeOut = 60000)
```

Comment:
No need to add `source` to `packageSource`. It increases the footprint of the change without adding much in the clarity department.

Reply:
Just making it clear what `source` means here. It's public method that we are using it to write tests. So, it should be clear on what `source` is.